earth_enterprise/rpms/build.gradle

```diff
@@ -417,7 +429,8 @@ task openGeeServerRpm (type: GeeRpm, dependsOn: openGeeSharedFiles) {
     installUtils = file('build/shared/install-utils.sh')
 
     // Add geresetpgdb script:
-    installUtils = file(new File(stagedInstallDir_server_opt, 'google/bin/gepgcommon'))
+    installUtils = file( new File( working_dir + "server/gepgcommon" ) )
```

Comment:
I don't think this line needs to change. `stagedInstallDir_server_opt` is a directory under `/tmp/fusion_os_install`, which should be the same regardless of the version built.

Reply:
More information is needed on the file 'gepgcommon' and it's location.

Reply:
@tst-dmendyke: I don't understand your comment.  What information is needed about `gepgcommon`?